### PR TITLE
Read confirmation

### PR DESCRIPTION
**Pull request by aabb2002**
Message shows after saving read status.

### CODE REVIEW

**Comment by aabb2002** [comment]
accidently added the read and ui files. The only change was in the main file.

**Comment by claraj** [comment]
This has a lot of code from other issues. Make sure you check out main before creating a new branch. Check your other PRs in case they have the same issue? Suggest checking out main, make a new branch, copy only the work related to the read confirmation to the new branch, and make a new PR. Same for the other PRs that are affected

File: main.py [hunk 1]
```diff
@@ -65,6 +65,7 @@ def change_read():
         new_read = ui.get_read_value()
         book.read = new_read
         book.save()
+        ui.message('You have read ' + book.title + ' by ' + book.author)
```
Comment (Baldeaglefacts) [inline main.py]: This doesn't check if the person is changing the status from read to unread, or vice versa. I tested the code and changed a books status from "read" to "unread", and it responded by saying "You have read Test_Title by Test_Author". It should say the opposite.

**Review by Baldeaglefacts — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (aabb2002) [reply main.py]: Got it. Now the read status is checked first before giving a message.

**Review by Baldeaglefacts — APPROVED** [review]

**Comment by Baldeaglefacts** [comment]
Works perfectly

**Comment by claraj** [comment]
This shouldn't have been merged with the unrelated changes, see earlier note.

**Pull request merged**